docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+
+On the host:
+
+1. create a bond: 
+``` bash
+sudo ip link add bond0 type bond mode active-backup miimon 50 num_grat_arp 5 primary enp2s1
+```
+2. add the slaves to the bond:
+``` bash
+  sudo ip link set link enp2s1 master bond0
+  sudo ip link set link enp2s2 master bond0
+```
+3. Add an address to the bond:
+``` bash
+  sudo ip addr add 10.30.10.10/24 dev bond0`
+```
+4. Make the bond  the default route:
+```bash
+  sudo ip route add default via 10.30.10.1
+```
+
+View the link layer information for the interfaces to see that the same MAC
+address is on all 3 interfaces: bond0, enp2s1,enp2s2:
+``` console 
+core@server-1 ~ $ ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: enp2s1: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master bond0 state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff permaddr a0:36:9f:3f:1a:e8
+3: enp2s2: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master bond0 state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff permaddr 14:02:ec:7f:3a:3c
+4: enp2s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
+    link/ether 0c:20:12:fe:01:00 brd ff:ff:ff:ff:ff:ff
+10: bond0: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+13: bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+```
+
+Confirm that the bonding driver is reporting the selected primary interface:
+```console
+cat /proc/net/bonding/bond0
+
+Ethernet Channel Bonding Driver: v6.6.74-flatcar
+
+Bonding Mode: fault-tolerance (active-backup)
+Primary Slave: enp2s1 (primary_reselect always)
+Currently Active Slave: enp2s1
+MII Status: up
+MII Polling Interval (ms): 50
+Up Delay (ms): 0
+Down Delay (ms): 0
+Peer Notification Delay (ms): 0
+
+Slave Interface: enp2s1
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: a0:36:9f:3f:1a:e8
+Slave queue ID: 0
+
+Slave Interface: enp2s2
+MII Status: up
+Speed: 10000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 14:02:ec:7f:3a:3c
+Slave queue ID: 0
+```
+
+### Failure Scenarios
+When `miimon` detects that the primary slave is no longer `up`, the bonding
+driver will send 5 gratitious arps via the secondary slave to notify the switch
+that the IP address is now on a new switch. The fabric will start routing
+traffic to this location and the host will start receiving traffic.
+
+If the `miimon` polling determines that the primary slave link is `up` the
+bonding driver will switch back to the primary slave. This behavior is
```

Comment:
```suggestion
If the `miimon` polling determines that the primary bonded interface link is `up` the
bonding driver will switch back to this primary bonded interface. This behavior is
```

Let's try to avoid the “master” and “slave” terminology if we can, please?